src/ansys/geometry/core/tools/problem_areas.py

```diff
@@ -237,15 +303,17 @@ def fix(self) -> RepairToolMessage:
             return RepairToolMessage(False, [], [])
 
         parent_design = get_design_from_edge(self.edges[0])
+
         response = self._repair_stub.FixInexactEdges(
             FixInexactEdgesRequest(inexact_edge_problem_area_id=self._grpc_id)
         )
-        parent_design._update_design_inplace()
+
```

Comment:
After the fix operation... we are not updating the design? This seems problematic